customTests/azure_gpu_vbios.nhc

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+function check_vbios_version() {
```

Comment:
Is this test actually used?

Reply:
Open to a discussion about it. During the thermal experiment it proved instrumental in diagnosing a collection of nodes missing essential CRD updates. However, it requires knowing precisely what VBIOS version a given cluster of nodes should be deployed with. The reason I left it but commented out (have since removed it per Jithin's comment) is that for general purpose use it is a much more difficult problem to know what the expected VBIOS version should be at a given time and we can expect it to change over time.

We have plans to run experiments on another cluster where we would use distributed NHC to prune the nodes, in this case we would add the VBIOS check back in pinned to the VBIOS version we expect from that cluster. Leaving it in provides easy access to the test that we got a lot of value out of.